public/API/API_GetGameInfoAndUserProgress.php

```diff
@@ -19,6 +19,7 @@
  *  int        NumAwardedToUserHardcore   number of achievements earned by the user in hardcore
  *  string     UserCompletion             percentage of achievements earned by the user
  *  string     UserCompletionHardcore     percentage of achievements earned by the user in hardcore
+ *  int        TotalPlaytime              total time the user has spent playing the game, in seconds
```

Comment:
Recommend using `UserTotalPlaytime` for consistency with `UserCompletion` and `UserCompletionHardcore` to better stress that this metric is user specific.

Although, `HighestAwardKind` and `HighestAwardDate` don't do that. 

@wescopeland, opinions?

Reply:
`UserTotalPlaytime` feels contextually appropriate for this endpoint.

Reply:
renamed